components/scrape_it_cloud/actions/start-scraping/start-scraping.mjs

```diff
@@ -0,0 +1,91 @@
+import app from "../../scrape_it_cloud.app.mjs";
+import utils from "../../common/utils.mjs";
+
+export default {
+  key: "scrape_it_cloud-start-scraping",
+  name: "Start Scraping",
+  description: "Initiate the scraping process for a specific endpoint. [See the documentation here](https://scrape-it.cloud/docs/api-features/basic-request).",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    app,
+    url: {
+      type: "string",
+      label: "URL",
+      description: "Specify the URL of the web page you would like to scrape.",
+    },
+    headers: {
+      type: "object",
+      label: "Headers",
+      description: "Specify custom HTTP headers to be passed to the request. For example, in order to override the User-Agent header, use `User-Agent` as a key and `Teapot` as a value.",
+      optional: true,
+    },
+    blockResources: {
+      type: "boolean",
+      label: "Block Resources",
+      description: "By default Scrape-it.cloud do not block images and CSS in the scraped page. To speed up requests and block images and CSS set this parameter value to true.",
```

Comment:
```suggestion
  description: "By default, Scrape-it.cloud doesn't block images and CSS in the scraped page. To speed up requests and block images and CSS, set this parameter value to true."
```

Reply:
Changed "do not" to "doesn't" and removed unnecessary hyphen as per Google's Developer docs style guide.